tests/coverage/test

```diff
@@ -1,51 +1,55 @@
 #!/bin/bash -e
-. "$(dirname "$0")"/../common.sh
 
-bazel_bin=$(bazel info bazel-bin)
+# TODO(Andy Scott): Reenable this test after fixing coverage in Bazel 25
```

Comment:
@andyscott @jjudd any idea how to fix coverage? I've tried to get this file working, and managed to make a bit of progress with
```diff
diff --git a/rules/common/private/utils.bzl b/rules/common/private/utils.bzl
index b6be98a..ffe03e0 100644
--- a/rules/common/private/utils.bzl
+++ b/rules/common/private/utils.bzl
@@ -83,7 +83,7 @@ def write_launcher(
         metadata_file = ctx.actions.declare_file("%s.jacoco_metadata.txt" % ctx.attr.name, sibling = output)
         ctx.actions.write(metadata_file, "\n".join([
             jar.short_path.replace("../", "external/")
-            for jar in jacoco_classpath
+            for jar in jacoco_classpath.to_list()
         ]))
         more_outputs = [metadata_file]
         more_substitutions = {
diff --git a/rules/private/phases/phase_test_launcher.bzl b/rules/private/phases/phase_test_launcher.bzl
index d0d107b..15f1bce 100644
--- a/rules/private/phases/phase_test_launcher.bzl
+++ b/rules/private/phases/phase_test_launcher.bzl
@@ -25,7 +25,7 @@ def phase_test_launcher(ctx, g):
             ctx,
             base = g.coverage.replacements,
         ).replacements
-        coverage_runner_jars = depset(direct = [ctx.files._jacocorunner + ctx.files._lcov_merger])
+        coverage_runner_jars = depset(direct = ctx.files._jacocorunner + ctx.files._lcov_merger)
 
     test_jars = depset(direct = [
         coverage_replacements[jar] if jar in coverage_replacements else jar
```
(by "a bit of progress" I mean it now fails during the execution step instead of during analysis). But I am now getting `java.lang.NoClassDefFoundError: org/jacoco/agent/rt/internal_b0d6a23/Offline` which doesn't have as obvious of a fix. Figured I'd ask if anyone had any ideas before I banged my head against a wall repeatedly